designer/client/field-edit.tsx

```diff
@@ -140,17 +140,16 @@ export function FieldEdit() {
               className="govuk-label govuk-checkboxes__label"
               htmlFor="field-options-required"
             >
-              {i18n("fieldEdit.optional", { fieldTitle })}
+              {i18n("common.componentOptionalOption.title", {
```

Comment:
This change conflicts with other tickets where field title appears in the string (e.g: TICKET: FORMS-234), reason for the `{i18n("fieldEdit.optional", { fieldTitle })}`, could you confirm with Haymon?

Reply:
Spoke to Haymon, he said the changes in 214 were about removing a field. He said this is correct.